flocker/restapi/_infrastructure.py

```diff
@@ -58,16 +74,10 @@ def _logging(original):
     """
     @wraps(original)
     def logger(self, request, **routeArguments):
-        try:
-            logger = self.logger
-        except AttributeError:
-            logger = _logger
-        else:
-            if logger is None:
-                logger = _logger
+        logger = _get_logger(self)
 
-        path = repr(request.path).decode("ascii")
```

Comment:
This line looked stupid, I guess.  The point was to handle non-ascii request paths though (I think).  I don't think this change is particularly for the better.  At the very least it seems worth a separate issue and some new tests.